Makefile

```diff
@@ -0,0 +1,3 @@
+
+build:
+	docker compose build chatserver
```

Comment:
`Makefile`, nice; I like this a lot and have been meaning to start using them for Go projects.